spec/models/piece_spec.rb

```diff
@@ -15,7 +15,11 @@
         password: 'foobar',
         password_confirmation: 'foobar'
       )
-      @game = @user.games.create(white_player_id: @user)
+      @game = @user.games.create(white_player_id: @user.id)
+      FROM_ROW = 3
```

Comment:
I just tried to do that. 
I have @user = ... and @game = ... after describe "#move_to" and before the "before(:all)
now every @game.pieces.create fails for pieces nil:NilClass.